addons/web/static/src/js/views/view_dialogs.js

```diff
@@ -314,6 +293,15 @@ var SelectCreateDialog = ViewDialog.extend({
             event.stopPropagation();
             this.$footer.find(".o_select_button").prop('disabled', !event.data.selection.length);
         },
+        field_changed: function (ev) {
```

Comment:
Do you remember why?

Reply:
It was asked by @ged-odoo to remove it from ```SelectCreateDialog``` for safety measure

Reply:
@msh-odoo In this case,  no `field_changed` event can arrive here, since these events are stopped by field manager mixins, and there is a sub field manager mixin (a `ListController`).  So, these 3 lines are not useful

Reply:
Also, I think that stopping `switch_view` and `do_action` make sense, but I would only do it if we can reproduce a problem, and in that case, it should come with a test and in a separate commit.

Having many small commits with a fix, a test, and an explanation is very useful :)

Reply:
@ged-odoo tests are moved in enterprise as Many2one field which has code related to mobile is moved in enterprise, some commits will be squashed once review is done